src/main/java/com/Example/BankManagement/Repository/AccountRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.Example.BankManagement.Repository;
+
+import com.Example.BankManagement.Entity.Account;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface AccountRepository  extends JpaRepository<Account, Long> {
+    Account getAccountDetailsBycustomerid(int customerid);
```

Comment:
if a customer has more than one account how it will handle?

Reply:
Right now we are returning Account type data if we want data of more then one account for single customer , then we have to return List of Account